prodigy/web/web_equivalence.py

```diff
@@ -160,7 +160,7 @@ def analyze_raw_code():
         app.logger.info(
             "Analysis task started for %s with input distribution %s", program,
             input_dist)
-        result, error_prob = compute_discrete_distribution(program, input_dist, config)
+        result, _ = compute_discrete_distribution(program, input_dist, config)
```

Comment:
does this mean that the web interface can only handle distributions which are already normalised?

Reply:
I've only removed this variable because lint was complaining that it was never used. Same as above.